frontend/webapp/views/main/templates/hijacks.htm

```diff
@@ -660,16 +679,37 @@ <h1>Additional actions</h1>
                     table.draw();
                     }
                 );
-                {% if current_user.has_roles('admin') %}
-
-
-                {% endif %}
 
                 $('#timezone').html(display_timezone());
                 displayHelpTextButton();
                 $("#navbar_hijacks").addClass("active");
             });
 
+            {% if current_user.has_roles('admin') %}
```

Comment:
question: a user cannot simply change this client-side and be able to send queries via Flask?

Reply:
Even if he guess 'where' to send 'what', @roles_required("admin") in Flask should block the message.